src/components/DatePicker.js

```diff
@@ -0,0 +1,454 @@
+
+    while (startDate.isBefore(endDate)) {
+      const isCurrentMonth = startDate.month() === currentDate.month();
+      const isCurrentDay = startDate.format(this.props.format) === selectedDate.format(this.props.format);
+      const day = startDate.isBefore(minimumDate) ? (
+        <div
+          key={startDate.month() + '-' + startDate.date()}
+          style={styles.calendarDay}
+        >
+          <div
+            key={startDate.format('DDDD')}
+            style={[styles.calendarDayContent, styles.calendarDayDisabled]}
+          >
+            {startDate.date()}
+          </div>
+        </div>
+      ) : (
+        <div
+          key={startDate.month() + '-' + startDate.date()}
+          onClick={this._handleDateSelect.bind(this, startDate.unix())}
+          style={[
+            styles.calendarDay,
+            isCurrentMonth && styles.currentMonth
+          ]}
+        >
+          <div
+            key={startDate.format('DDDD')}
+            style={[styles.calendarDayContent, isCurrentDay && styles.currentDay]}
+          >
+            <div style={styles.calendarDayText}>{startDate.date()}</div>
+          </div>
+        </div>
+      );
+
+      days.push(day);
+      startDate.add(1, 'd');
+    }
+
+    //Divide calendar columns/Rows
+    if (this.props.divideCells) {
+      for (let i = 0; i < days.length; i++) {
+        if (this._shouldBorderCell(i + 1)) {
+          days[i].props.style.push(styles.borderRight);
+        }
+
+        if (this._shouldBorderRow(i + 1, days.length + 1)) {
+          days[i].props.style.push(styles.borderBottom);
+        }
+      }
+    }
+
+    return days;
+  }
+
+  _renderScrim (styles) {
+    if (this.props.useScrim && this.state.showCalendar) {
+      return (
+        <div style={[styles.scrim, this.props.scrimStyle]}/>
+      );
+    }
+  }
+
+  _renderSelectedDate (styles) {
+    if (this.props.useInputForSelectedDate) {
+      return (
+        <div>
+          <input
+            key='input'
+            onClick={this._toggleCalendar.bind(this)}
+            style={styles.input}
+            type='text'
+            value={moment.unix(this.props.selectedDate).format(this.props.format)}
+          />
+          <Icon
+            onClick={this._toggleCalendar.bind(this)}
+            size='1.75em'
+            style={styles.calendarIcon}
+            type={this.state.showCalendar ? 'caret-up' : 'caret-down'}
+          />
+        </div>
+      );
+    } else {
+      return (
+        <div
+          key='selectedDate'
+          onClick={this._toggleCalendar.bind(this)}
+          style={styles.selectedDate}
+        >
+          {moment.unix(this.props.selectedDate).format(this.props.format)}
+        </div>
+      );
+    }
+  }
+
+  _renderTitle (styles) {
+    if (this.props.title) {
+      return (
+        <div key='title' style={styles.title}>
+          {this.props.title}
+        </div>
+      );
+    } else {
```

Comment:
You should be able to remove this `else` statement all together. If there is no title, then we don't return anything.